kafka-admin/src/main/java/org/bf2/admin/kafka/admin/handlers/OASModelFilter.java

```diff
@@ -73,6 +73,11 @@ public void filterOpenAPI(OpenAPI openAPI) {
 
         // Sort global schemas
         openAPI.getComponents().setSchemas(new TreeMap<>(openAPI.getComponents().getSchemas()));
+        var info = openAPI.getInfo();
+        
+        info.setTitle("Kafka Instance API");
+        info.setDescription("API for interacting with Kafka Instance. Includes Produce, Consume and Admin APIs");
+        info.setContact(OASFactory.createContact().name("Support email").email("rhosak-support@redhat.com"));
```

Comment:
@wtrocki - we may ultimately need to get rid of this email in the upstream repo.

Reply:
Is the contact information used in the SDKs or only intended for Swagger UI? If only Swagger UI, we can remove this and patch it in later in our CI process.

Reply:
I will remove it now.